Addon.Episerver.EnvironmentSynchronizer/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Addon.Episerver.EnvironmentSynchronizer.DynamicData;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Addon.Episerver.EnvironmentSynchronizer.Configuration
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddEnvironmentSynchronization(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddSingleton<IConfigurationReader, ConfigurationReader>();
```

Comment:
If the dependencies are added this way, one should drop the Serviceconfiguration-attributes, right? 
[ServiceConfiguration(typeof(IConfigurationReader))]